websocket_server/ApiService.js

```diff
@@ -7,16 +7,12 @@
 
 import fetch from 'node-fetch'
 import https from 'https'
-import dotenv from 'dotenv'
-import Utils from './Utils.js'
-dotenv.config()
+import Config from './Config.js'
 
 export default class ApiService {
 
 	constructor(tokenGenerator) {
-		this.NEXTCLOUD_URL = process.env.NEXTCLOUD_URL
-		this.IS_DEV = Utils.parseBooleanFromEnv(process.env.IS_DEV)
-		this.agent = this.IS_DEV ? new https.Agent({ rejectUnauthorized: false }) : null
+		this.agent = (Config.USE_TLS && Config.BYPASS_SSL_VALIDATION) ? new https.Agent({ rejectUnauthorized: false }) : null
```

Comment:
Shouldn't BYPASS_SSL_VALIDATION rather only change the rejectUnauthorized value?
```suggestion
		this.agent = (Config.USE_TLS) ? new https.Agent({ rejectUnauthorized: !Config.BYPASS_SSL_VALIDATION }) : null
```